aten/src/THC/THCAsmUtils.cuh

```diff
@@ -106,8 +104,8 @@ __device__ __forceinline__ unsigned getLaneMaskLt() {
 }
 
 #if defined (__HIP_PLATFORM_HCC__)
-__device__ __forceinline__ unsigned long long int getLaneMaskLe() {
-  std::uint64_t m = (1ull << (getLaneId() + 1ull)) - 1ull;
+__device__ inline unsigned long long int getLaneMaskLe() {
```

Comment:
Given how `__forceinline__` is now correctly defined, I'd like to keep it that way